.kitchen.yml

```diff
@@ -2,35 +2,33 @@ driver:
   name: vagrant
 
 driver_config:
-  require_chef_omnibus: true
+  require_chef_omnibus: latest
 
 provisioner:
   name: chef_zero
 
 platforms:
-- name: ubuntu-14.04
-  run_list:
-  - recipe[apt]
-
-- name: ubuntu-12.04
-  run_list:
-  - recipe[apt]
-
-- name: ubuntu-10.04
-  run_list:
-  - recipe[apt]
-
-- name: centos-6.5
-
-- name: centos-7.0
+  - name: ubuntu-14.04
+    run_list:
+      - recipe[apt]
+  - name: ubuntu-12.04
+    run_list:
+      - recipe[apt]
+  - name: ubuntu-10.04
+    run_list:
+      - recipe[apt]
+  - name: centos-6.6
```

Comment:
I upgraded Centos versions since those are the latest referenced in Chef's Bento page: https://github.com/chef/bento

I can't confirm any kitchen tests because there are none :-) I'd like to add ChefSpec examples next.